.eslintrc

```diff
@@ -0,0 +1,47 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    // "plugin:prettier/recommended",
```

Comment:
Note: To enable `prettier`, we need to uncomment this line and remove the `"requirePragma": true` from `.prettierrc`